src/App.js

```diff
@@ -1,13 +1,67 @@
 import React from 'react';
 import './App.css';
+import Button from "./components/Button/Button";
+import Product from "./components/Producten/Product";
+import Tile from "./components/Tile/Tile";
+import afbeelding1 from "./assets/bag_1.png";
+import afbeelding2 from "./assets/bag_2.png";
+import afbeelding3 from "./assets/bag_3.png";
+import afbeelding4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import ourStory from "./assets/our_story.png";
 
 function App() {
   return (
+      <>
       <h1>Handbags & Purses</h1>
-  );
-}
+          <nav>
+              <Button text="to the collection" />
+              <Button text="shop all bags" />
+              <Button text="pre-orders" />
+          </nav>
+          <main>
+            <article>
+                <Product label="Best seller" afbeelding={afbeelding1} afText="bag_1" text="the handy bag"
```

Comment:
Het is leesbaarder wanneer je de verschillende properties onder elkaar zet, zie ook de uitwerking.